### DISCUSSION TITLE
Combining useQueries() with infinite queries?

### DISCUSSION THREAD

**Discussion by holtgrewe** [TanStack/query#7991] [Q&A] [answered]
Hello,

today I had the use case where I needed to launch a paginated search for multiple items in a list. I did not get `useInfiniteQuery()` to run inside a `computed(() => {})`. Is there a way to execute `useInfiniteQuery()` style queries within `useQueries()`?

I'm using Vue Query.

Thanks!

**Comment by TkDodo** [comment, marked as answer]
no, we don't have a way to call an arbitrary amount of inifinite queries as a single hook.

**Reply by holtgrewe** [reply]
Thanks for the quick reply!
